dbt/tests/generic/test_expression_is_true.sql

```diff
@@ -4,10 +4,10 @@
 -- an optional `select_columns` option representing an array of columns to
 -- select for failing rows. If no `select_columns` array is provided, defaults
 -- to selecting 1 for failing rows.
-{% test expression_is_true(model, expression, select_columns=[1]) %}
+{% test expression_is_true(model, expression, column_name, select_columns=[1]) %}
     {%- set columns_csv = select_columns | join(", ") %}
 
     select {{ columns_csv }}
     from {{ model }}
-    where not ({{ expression }})
+    where not ({{ column_name }} {{ expression }})
```

Comment:
I added this in order to use `expression_is_true` in column-level tests, i.e.:

```yaml
          - name: rmbed
            tests:
              - expression_is_true:
                  name: iasworld_dweldat_rmbed_lte_rmtot
                  expression: <= rmtot
                  select_columns:
                    - parid
                    - taxyr
                    - card
```

It still works fine in the original context of passing a full expression.